futures.js

```diff
@@ -12,7 +12,7 @@ const S             = require('./common/sanctuary');
 
 //    readFile :: String -> Future Error String
 const readFile =
-S.compose(Future.node, S.curry3(fs.readFile, S.__, {encoding: 'utf8'}));
+S.curry3(Future.encaseN2, fs.readFile, S.__, {encoding: 'utf8'});
```

Comment:
I find the placeholder clearer than flipping in this case. I'm happy to change it, though.